addon/-microstate.js

```diff
@@ -35,7 +50,7 @@ export default Ember.Helper.extend({
             return microstate.transition(key, ()=> action.call(null, ...context, ...args));
           };
         } else {
-          let next = object[key];
+          let next = object[`_${key}`];
```

Comment:
Once we've implemented object enumerability as a hook on the `Object` microstate, let's re-evaluate if this is necessary. If so, let's find out why.